server/web/config.go

```diff
@@ -58,6 +59,7 @@ func Config(c *gin.Context) {
 	c.Header("Content-Type", "text/javascript; charset=utf-8")
 	tmpl := template.Must(template.New("").Funcs(funcMap).Parse(configTemplate))
 
+	c.Status(http.StatusOK)
```

Comment:
I would add the status after the error handeling (raw http std would have an issue - nut sure if gin do ...)